cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e
+
+_cli_container="kubevirtci/gocli@sha256:868fd9f2b6e5ff6473ab92cdd7d3a762ec8a8c66dac29f5db841879d40038f2a"
+_cli_with_tty="docker run --privileged --net=host --rm -t -v /var/run/docker.sock:/var/run/docker.sock ${_cli_container}"
+_cli="docker run --privileged --net=host --rm ${USE_TTY} -v /var/run/docker.sock:/var/run/docker.sock ${_cli_container}"
+
+function _main_ip() {
+    echo 127.0.0.1
+}
+
+function _port() {
+    ${_cli} ports --prefix $provider_prefix "$@"
+}
+
+function prepare_config() {
+    BASE_PATH=${KUBEVIRTCI_CONFIG_PATH:-$PWD}
+    cat >$BASE_PATH/$KUBEVIRT_PROVIDER/config-provider-$KUBEVIRT_PROVIDER.sh <<EOF
+master_ip=$(_main_ip)
+kubeconfig=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
+kubectl=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubectl
+gocli=${BASE_PATH}/../cluster-up/cli.sh
+docker_prefix=localhost:$(_port registry)/kubevirt
+manifest_docker_prefix=registry:5000/kubevirt
+EOF
+}
+
+function _registry_volume() {
+    echo ${job_prefix}_registry
+}
+
+function _add_common_params() {
+    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE} --cpu 5 --secondary-nics ${KUBEVIRT_NUM_SECONDARY_NICS} --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+    if [[ $TARGET =~ windows.* ]] && [ -n "$WINDOWS_NFS_DIR" ]; then
+        params=" --nfs-data $WINDOWS_NFS_DIR $params"
+    elif [[ $TARGET =~ os-.* ]] && [ -n "$RHEL_NFS_DIR" ]; then
+        params=" --nfs-data $RHEL_NFS_DIR $params"
+    fi
+    echo $params
+}
+
+function _kubectl() {
+    export KUBECONFIG=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
```

Comment:
```
_kubectl create ns kubevirt-hyperconverged
+ export KUBECONFIG=/go/src/github.com/kubevirt/hyperconverged-cluster-operator/_ci-configs/k8s-1.13.3/.kubeconfig
+ KUBECONFIG=/go/src/github.com/kubevirt/hyperconverged-cluster-operator/_ci-configs/k8s-1.13.3/.kubeconfig
+ /go/src/github.com/kubevirt/hyperconverged-cluster-operator/_ci-configs/k8s-1.13.3/.kubectl create ns kubevirt-hyperconverged
/go/src/github.com/kubevirt/hyperconverged-cluster-operator/cluster-up//cluster/ephemeral-provider-common.sh: line 45: /go/src/github.com/kubevirt/hyperconverged-cluster-operator/_ci-configs/k8s-1.13.3/.kubectl: No such file or directory 
```

Prow job is failing here because `${CMD} create ns kubevirt-hyperconverged` is translated to `_kubectl create ns kubevirt-hyperconverged`